components/devtools/Cargo.toml

```diff
@@ -11,6 +11,7 @@ name = "devtools"
 path = "lib.rs"
 
 [dependencies]
```

Comment:
Please remove `time` from the dependencies as well. It looks like there is one more use in `network_event.rs` before that's possible.

Reply:
removed it